src/css/components/menu/_menu-popup.scss

```diff
@@ -19,6 +19,20 @@
   max-height: 15em;
 }
 
+.vjs-layout-small .vjs-menu-button-popup .vjs-menu .vjs-menu-content {
```

Comment:
should we add `x-small` and `tiny` here? Or maybe just have those together be something like 5em?

Reply:
I didn't add it because the player hides captions for x-small and tiny layouts but maybe we should have some default value for all layouts. What do you think?

Reply:
Probably worth having it in case someone forces the menu buttons on.

Reply:
Ok, I added missing `tiny` and `x-small` classes.

Reply:
Thanks!

Reply:
np